hal/lib/athena/FPGAEncoder.cpp

```diff
@@ -34,17 +34,17 @@ HAL_FPGAEncoderHandle HAL_InitializeFPGAEncoder(
     HAL_Handle digitalSourceHandleB, HAL_AnalogTriggerType analogTriggerTypeB,
     HAL_Bool reverseDirection, int32_t* index, int32_t* status) {
   bool routingAnalogTriggerA = false;
-  uint8_t routingPinA = 0;
+  uint8_t routingChannelA = 0;
   uint8_t routingModuleA = 0;
-  bool successA =
-      remapDigitalSource(digitalSourceHandleA, analogTriggerTypeA, routingPinA,
-                         routingModuleA, routingAnalogTriggerA);
+  bool successA = remapDigitalSource(digitalSourceHandleA, analogTriggerTypeA,
+                                     routingChannelA, routingModuleA,
+                                     routingAnalogTriggerA);
```

Comment:
I'm not sure why this got reformatted?  What version of clang-format is being used?